src/utils/hooks/auth/useEmailVerification.ts

```diff
@@ -0,0 +1,113 @@
+import useBooleanState from 'utils/hooks/state/useBooleanState';
+import { type InputMessage } from 'pages/Auth/SignupPage/components/CustomInput';
+import useCountdownTimer from 'pages/Auth/SignupPage/hooks/useCountdownTimer';
+import {
+  emailExists, verificationEmailSend, verificationEmailVerify, idFindEmail,
+} from 'api/auth';
+import { MESSAGES } from 'static/auth';
+import { isKoinError } from '@bcsdlab/koin';
+import ROUTES from 'static/routes';
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+import { useMutation } from '@tanstack/react-query';
+
+function usePhoneVerification({ email }: { email: string }) {
```

Comment:
```suggestion
function useEmailVerification({ email }: { email: string }) {
```
네이밍이 수정되어야할 것 같은데 맞나요?